docs/api/create_docker_container.md

```diff
@@ -0,0 +1,78 @@
+# Creating a container
+
+Testcontainers' generic container support offers the most flexibility, and makes it easy to use virtually any container images as temporary test environments. To interact or exchange data with a container, Testcontainers for .NET provides an implementation of the `TestcontainersBuilder<TDockerContainer>` to configure and create the resource.
```

Comment:
```suggestion
Testcontainers' generic container support offers the greatest flexibility and makes it easy to use virtually any container image in the context of a temporary test environment. To interact or exchange data with a container, Testcontainers provides  `TestcontainersBuilder<TDockerContainer>` to configure and create the resource.
```

Remove the `for .NET` part here, since back in the discussion we assumed, within an established context, we can use Testcontainers directly. This messaging is likely not yet consistent across the docs, so was thinking we try it out here. 

Open for discussion.